chainerx_cc/.clang-tidy

```diff
@@ -4,15 +4,17 @@ bugprone-*,\
 cert-*,\
 cppcoreguidelines-*,\
 -cppcoreguidelines-pro-bounds-pointer-arithmetic,\
+-cppcoreguidelines-pro-type-union-access,\
 clang-analyzer-*,\
 google-*,\
 -google-runtime-references,\
 misc-*,\
+-misc-macro-parentheses,\
```

Comment:
For use of kernel class as argument without parenthesis in (`.. : public key_kernel_cls`) in kernel registration macro.

Reply:
How about explicitly write `NOLINT`?
This warning is actually useful in usual cases.

Reply:
I agree, but do you know how to do that in a multi-line macro?

Reply:
I think `/* */` works (as done in `CHAINERX_NEVER_REACH`).

Reply:
Perhaps in combination with `NOLINTNEXTLINE`.

Reply:
Ah, thanks a lot that worked. Applied your suggested change to appropriate places and removed the whitelist entry.